packages/ui/src/examples/E2_FetchWithStrapiRequest.tsx

```diff
@@ -1,17 +1,51 @@
-/* eslint-disable @typescript-eslint/no-unused-vars */
-import { useEffect, useState } from 'react'
+import React, { useEffect, useState } from 'react';
 
-import { Box } from '@chakra-ui/react'
+import { Box } from '@chakra-ui/react';
 
-import { strapiRequest } from '@wsvvrijheid/lib'
+import { strapiRequest } from '@wsvvrijheid/lib';
+
+const PUBLIC_TOKEN = '2ff0613e2f32eae88e6dbdb8f806d8b1bdbb7fab7b1b9c2f693013220fceae7423a1eb8fdf06938c3bd1afd8e695364e883434d714b223b0a2fb585fcd06432f105d80ff90be73f59d2fc4690919c8737e75b0f786b19f4cc6454e638ea77cefe3675d592580eb77656dcfcdb8c1042d599313e297012703ebf3ecf1d6e30603';
+
+// const blogUrl = `https://wsvv-api-staging.onrender.com/api/blogs?locale=tr` -> I don't know how the implement this url to the strapiRequest function. Beside this problem I think code is working properly.
+
+interface Blog {
+  id: number;
+  title: string;
+}
 
 export const FetchWithStrapiRequest = () => {
-  const [blogs, setBlogs] = useState([])
+  const [blogs, setBlogs] = useState<Blog[]>([]);
 
   useEffect(() => {
-    // TODO: fetch blogs with strapiRequest by using the API_URL and PUBLIC_TOKEN
-    // Remember that fetcher is a wrapper around axios that adds the token and api url to the request
-  }, [])
+    const fetchData = async () => {
+      try {
+        // I couldn't find any argument for api url in strapiRequest function
+        const response = await strapiRequest({
```

Comment:
As the name suggests, `strapiRequest` is simply an axios wrapper for fetching data from our Strapi backend. It's a generic function (`function<T> {}`) that allows you to specify which type of Strapi model _(Blog, Activity, Hashtag etch)_ you want to retrieve data from.

```suggestion
        const response = await strapiRequest<Blog>({
```